### DISCUSSION TITLE
抓到一个抹茶小蛋糕（嘻嘻嘻）

### DISCUSSION THREAD

**Discussion by Rolesican** [share121/inter-knot#2642] [其他]
<img src="https://bbs-static.miyoushe.com/static/2024/03/21/dab9bdd5012200a2902c8ae19dc1c8c9_4053703670259707891.png" alt="丽都漫步-可琳" width="75" ><img src="https://bbs-static.miyoushe.com/static/2024/03/21/dab9bdd5012200a2902c8ae19dc1c8c9_4053703670259707891.png" alt="丽都漫步-可琳" width="75" >
![20240815230544](https://github.com/user-attachments/assets/88bbb155-c23d-4324-9e2c-2c477c937974)

**Comment by qmtqyh** [comment]
一口吃掉！(啊呜

**Comment by zzz-Rope** [comment]
我要了<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Comment by cyberrobber** [comment]
好可爱

**Comment by QingMingqwq** [comment]
可爱捏

**Comment by Natsumowari** [comment]
可琳可爱

**Comment by scp-perseus** [comment]
可琳锯锯我的

**Reply by realyniger** [reply]
不是哥们

**Comment by bangbu01** [comment]
嗯呢！(可爱！)
![01](https://github.com/user-attachments/assets/54f528fd-f182-4836-8011-8532e196315a)

**Comment by ace0beind** [comment]
<img src="https://bbs-static.miyoushe.com/static/2023/11/16/2544d70667e060623d4b7702cd6bb3b9_8763505029750571668.png" alt="可琳-紧张" width="75" >
